docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/sparse/fill-empty-rows-16.rst

```diff
@@ -0,0 +1,107 @@
+FillEmptyRows
+======================
+
+
+.. meta::
+  :description: Learn about FillEmptyRows-16 - a sparse operation, which
+                can be performed on four required input tensors.
+
+**Versioned name**: *FillEmptyRows-16*
+
+**Category**: *Sparse*
+
+**Short description**: Fills empty rows of an input sparse tensor with a default value.
+
+**Detailed description**:
+
+Operation FillEmptyRows is an implementation of ``tf.sparse.fill_empty_rows``.
+
+The input sparse tensor is represented by the three inputs: 
+
+* ``indices``
+* ``values``
+* ``dense_shape``
+
+For each row in the input sparse tensor, this operator checks if the row is empty. If the row is empty, the operator adds an entry with the specified default value at index `[row, 0, ..., 0]`. The input may have empty columns at the end, which will not be affected by this operation.
+
+The output sparse tensor will be in row-major order and will have the same shape as the input, but with updated `output_indices` and `output_values`.
+
+This operator also returns a boolean vector indicating which rows were filled with the default value: ``empty_row_indicator[i] = True`` if row ``i`` was an empty row.
+
+**Attributes**: FillEmptyRows-16 operation has no attributes.
+
+**Inputs**:
+
+* **1**: ``default_value`` a scalar of type *T* to be inserted into the empty rows. **Required.**
+* **2**: ``values`` 1D tensor containing the values of type *T* to be inserted at the specified indices. **Required.**
+* **3**: ``dense_shape`` 1D tensor of type *T_IDX* indicating the shape of the dense tensor. **Required.**
+* **4**: ``indices`` 2D tensor of type *T_IDX* indicating the positions at which ``values`` are placed in the sparse tensor. **Required.**
+    It is of shape ``[M, N]``, where:
+
+    * ``M`` is the same as the length of the ``values`` input.
+    * ``N`` is equal to the rank of ``dense_shape``.
+
+**Outputs**:
+
+* **1**: ``output_indices`` 2D tensor of type *T_IDX* indicating the positions at which ``output_values`` are placed in the sparse tensor.
+    It is of shape ``[M', N]``, where:
+
+    * ``M'`` is the length of the updated ``output_values``.
+    * ``N`` is equal to the rank of ``dense_shape``.
+* **2**: ``output_values`` 1D tensor containing the values of type *T* to be inserted at the specified indices.
+* **3**: ``empty_row_indicator`` 1D tensor of type ``boolean`` indicating True for rows which were empty before executing the operation.
```

Comment:
To have the whole sparse tensor representation, shouldn't the `dense_shape` be an output as well?
On the other hand, assuming it's the same as the input `dense_shape` maybe it's reduntant.

Reply:
`tf.raw_ops.SparseFillEmptyRows` doesn't return the output `SparseTensor` at all according to https://www.tensorflow.org/api_docs/python/tf/raw_ops/SparseFillEmptyRows#returns. Do you think we should have it?

That's also interesting because there's `tf.sparse.fill_empty_rows` and then there's `tf.raw_ops.SparseFillEmptyRows`, which have quite different outputs:

https://www.tensorflow.org/api_docs/python/tf/sparse/fill_empty_rows
https://www.tensorflow.org/api_docs/python/tf/raw_ops/SparseFillEmptyRows

However the BST model I received is using the `raw_ops` one and in the graph the `SparseTensor` is not returned.